packages/commonjs/src/transform.js

```diff
@@ -146,10 +146,28 @@ export function transformCommonjs(
   function isRequireStatement(node) {
     if (!node) return false;
     if (node.type !== 'CallExpression') return false;
-    if (node.callee.name !== 'require' || scope.contains('require')) return false;
-    // Weird case of require() without arguments
-    if (node.arguments.length === 0) return false;
-    return true;
+
+    const { callee } = node;
+
+    if (callee.type === 'Identifier' && callee.name === 'require') {
```

Comment:
It would be nice to have comments here to indicate the `require()` form that is checked :smile: 

```js    
    if (callee.type === 'Identifier' && callee.name === 'require') {
        // module.require()
```

```js
    } else if (callee.type === 'MemberExpression') {
        // require()
```

Reply:
great suggestion. @danielgindi could you add those in real quick?

Reply:
Done :-)